web/src/engine/osk/src/views/touchEventPromiseMap.ts

```diff
@@ -13,10 +13,10 @@ export default class TouchEventPromiseMap {
   }
 
   public maintainTouches(list: TouchList) {
-    let keys = [].concat(this.map.keys());
+    let keys = Array.from(this.map.keys());
```

Comment:
Does #11732 have any impact on this?

Reply:
This is outside of the worker.

We keep es6-shim around on Android and use it for devices with outdated WebViews; it'll cover this case well enough.